tests/test_main.py

```diff
@@ -161,7 +161,7 @@
     base_knights_config["arthur"]["potion"] = {
         "name": "Dragon's heart",
         "effect": {
-            "protection": +20,
+            "add_some_abilities": +20,
```

Comment:
Ensure that the key 'add_some_abilities' is used consistently across the codebase. If it is a typo or inconsistency, it should be corrected to match the expected key in the battle function.